doc/rados/operations/pools.rst

```diff
@@ -160,30 +155,29 @@ Where:
 
 .. describe:: --autoscale-mode=<on,off,warn>
 
-   If you set the autoscale mode to ``on`` or ``warn``, you can let the system
-   autotune or recommend changes to the number of placement groups in your pool
-   based on actual usage.  If you leave it off, then you should refer to
-   :ref:`placement groups` for more information.
+   - ``on``: the Ceph cluster will autotune or recommend changes to the number of PGs in your pool based on actual usage.
+   - ``warn``: the Ceph cluster will autotune or recommend changes to the number of PGs in your pool based on actual usage.
+   - ``off``: refer to :ref:`placement groups` for more information.
 
   :Type: String
   :Required: No.
-  :Default:  The default behavior is controlled by the :confval:`osd_pool_default_pg_autoscale_mode` option.
+  :Default: The default behavior is determined by the :confval:`osd_pool_default_pg_autoscale_mode` option.
 
 .. describe:: [expected-num-objects]
 
-   The expected number of objects for this pool. By setting this value (
-   together with a negative **filestore merge threshold**), the PG folder
-   splitting would happen at the pool creation time, to avoid the latency
-   impact to do a runtime folder splitting.
+   The expected number of objects for this pool. By setting this value and
+   assigning a negative value to **filestore merge threshold**, you arrange
```

Comment:
Filestore is an unbackend and should be removed from mention here

Reply:
Parsing this comment was like reading Finnegans Wake, but I finally cracked the code.

Reply:
This is going into Unfinished Business until I can find a suitable string to replace "filestore merge threshold".

Reply:
Paraphrasing 1984, _unperson_ etc.

Reply:
Yeah, yeah. I got it. I had to think of Stalin to get it. Then I got it.